Clarify documentation regarding installation media

Zugschlus opened the pull request:
This is the result of the discussion in
https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320

Commit:
Clarify documentation regarding installation media

This is the result of the discussion in
https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320

ekohl commented on _includes/manuals/1.17/4.4.2_prov_media.md:
```diff
@@ -12,3 +12,10 @@ Be sure to set the Family for the Media
+
+Most sites are well-served with having a 1:1 relationship between operating system and installation media. This will obviously work. However, there is a bunch of use cases that make sense to have more than one set of installation media per operating system. The most common case of this is when a site has datacenters in multiple locations and would like to have the systems automatically select the ...
+
+Another possible use case is to use differnet of installation media for staging of different operating system versions. This is, however significantly more bumpy since this raises a bunch of issues with the version field in the actual operating system and might need other adjustments in the OS deployment process since, for example, the files that are actually used for the tftp boot during network ...
```
s/differnet/different/

Zugschlus replied:
I'll file a new PR for the obvious one-letter change. Wouldn't it have been easier for all participants to simply merge and then do the one-letter commit?

ekohl commented on _includes/manuals/1.17/4.4.2_prov_media.md:
```diff
@@ -1,5 +1,5 @@
 
-The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set.
+The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set. In the Red Hat World, installation media are a resource for the installation repo.
```
This addition doesn't clarify it much for me.

lzap replied:
Yeah I'd drop this.

Zugschlus replied:
@lzap, the clarification was taken verbatim from your own suggestion in https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320, and it doesn't clarify much for me, but you said it was important to understand, so I put the text in my patch.

dLobatog reviewed (changes requested):
@Zugschlus Setting to waiting on contributor because it's missing a few things commented inline.

Commit:
fix trivial typo

Commit:
re-word installation/updates clarification to stay closed to original

(taken from
https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320)

Zugschlus commented:
I think that the required changes are done now, I don't know whether I can do anything to get rid of the "waiting for changes" status of the PR.

lzap commented on _includes/manuals/1.17/4.4.2_prov_media.md:
```diff
@@ -1,5 +1,5 @@
 
-The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set.
+The Installation Media represents the web URL from where the installation packages can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when Foreman is first installed. However, it is best to edit the media you are going to use and ensure the Family is set. It is important to understand that, in the Red Hat World, installation media are a resource for the installation repo,  ...
```
How about: Make sure to use URL with OS installer images, some Linux distributions publish repositories containing no installers (e.g. fedora-updates).

Zugschlus replied:
much better for me.

Zugschlus replied:
>The Installation Media represents the web URL from where the installation packages
>can be retrieved (i.e the OS mirror). Some OS Media is pre-created for you when
>Foreman is first installed. However, it is best to edit the media you are going to
>use and ensure the Family is set. Make sure to use URLs that point to
>actual OS installer repositories. The installer-less repositories published by
>some Linux distributions (e.g. fedora-updates) won't work here

how about that?

lzap replied:
Better, thanks. Also "However, it is best to edit the media you are going to
use and ensure the Family is set." is a bit weird to me. Perhaps "Be sure to select closest or local mirror and ensure family is set correctly."

Commit:
re-word again installation/updates clarification

    (taken from
    https://community.theforeman.org/t/need-use-case-for-different-installation-media-for-single-operating-system/8320)

GregSutcliffe commented:
+1 from me. @dLobatog  / @lzap is there anything else pending here?

lzap reviewed (approved):
No comments there, thanks.

GregSutcliffe commented:
Thanks @Zugschlus et al!
